modoboa/admin/management/commands/subcommands/_mx.py

```diff
@@ -79,6 +79,9 @@ def query_dnsbl(self, mx_list, provider):
             pattern = "{}.{}.".format(reverse, provider)
             try:
                 result = socket.gethostbyname(pattern)
+                #Mitigate Spamhaus status error (these codes don't mean ip is listed)
+                if result == "127.255.255.254" or result == "127.255.255.255":
```

Comment:
I think we should find something more generic to handle that...

Reply:
I've tried a more generic way as you proposed, tell me if that's not what you had in mind :)